projects/Monaliza.md

```diff
@@ -0,0 +1,90 @@
+## Project Name 
+
+Monaliza
+
+## Category 
+
+NFT, Truffle, NodeJS, React, Python, IPFS Storage, CELO, Docker, Kubernetes, Solidity
+
+## Project Description
+
+The project is aimed to be a connector between instagram/social media and NFTs, enabling the average instagram/social media user to be able to mint their own NFTs and submit them straight to their instagram, utilising their fanbase and followers, as well as Opensea, and persist this to IPFS as a storage point. This would hopefully get the average person to be able to adopt blockchain without having to leave the comfort of their favourite platform and make their instagram posts/pictures/videos an NFT.
+
+## Project Status
+
+Under Development
+
+## Previews
+
+![Landing Page Preview](/home/darien/Downloads/MonaSS.png)

```

Comment:
You need to upload to github - this path will not resolve.

Reply:
Hi, the path has now been edited too to resolve